latest/bpg/autoscaling/karpenter.adoc

```diff
@@ -112,7 +111,7 @@ Do not run Karpenter on a node that is managed by Karpenter.
 
 === No custom launch templates support with Karpenter
 
-There is no custom launch template support with v1beta1 APIs (v0.32+).
+There is no custom launch template support with v1 APIs.
```

Comment:
Let's add (v0.32+)
There is no custom launch template support with v1 APIs (v0.32+)